src/organizations/components/OrgNavigation.tsx

```diff
@@ -1,128 +1,80 @@
 // Libraries
-import React, {PureComponent} from 'react'
+import React, {FC} from 'react'
+import {useSelector} from 'react-redux'
 import {Link} from 'react-router-dom'
 
 // Components
 import {Tabs, Orientation, ComponentSize} from '@influxdata/clockface'
-import CloudExclude from 'src/shared/components/cloud/CloudExclude'
-import CloudOnly from 'src/shared/components/cloud/CloudOnly'
-import {FeatureFlag} from 'src/shared/utils/featureFlag'
 
 // Constants
-import {CLOUD_USERS_PATH, CLOUD_URL} from 'src/shared/constants'
-import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {CLOUD} from 'src/shared/constants'
 
-// Decorators
-import {ErrorHandling} from 'src/shared/decorators/errors'
+// Utils
+import {getOrg} from 'src/organizations/selectors'
 
 interface Props {
   activeTab: string
-  orgID: string
 }
 
 interface OrgPageTab {
   text: string
   id: string
-  cloudExclude?: boolean
-  cloudOnly?: boolean
-  href?: string
-  link?: string
-  featureFlag?: string
-  featureFlagValue?: boolean
+  enabled?: () => boolean
+  link: string
 }
 
-@ErrorHandling
-class OrgNavigation extends PureComponent<Props> {
-  public render() {
-    const {activeTab, orgID} = this.props
+const OrgNavigation: FC<Props> = ({activeTab}) => {
+  const orgID = useSelector(getOrg)?.id
 
-    // TODO(ariel): once the `unityUsers` flag goes live we won't need any of the href logic:
-    // https://github.com/influxdata/ui/issues/1405
-    const tabs: OrgPageTab[] = [
-      {
-        text: 'Members',
-        id: 'members-oss',
-        cloudExclude: true,
-        link: `/orgs/${orgID}/members`,
-      },
-      {
-        text: 'Users',
-        id: 'members-quartz',
-        cloudOnly: true,
-        link: isFlagEnabled('unityUsers') ? `/orgs/${orgID}/users` : null,
-        href: isFlagEnabled('unityUsers')
-          ? null
-          : `${CLOUD_URL}/organizations/${orgID}${CLOUD_USERS_PATH}`,
-      },
-      {
-        text: 'About',
-        id: 'about',
-        link: `/orgs/${orgID}/about`,
-      },
-    ]
+  const tabs: OrgPageTab[] = [
+    {
+      text: 'Members',
+      id: 'members-oss',
```

Comment:
tiny suggestion, only to nit: maybe consider pulling the ids into an enum at the top, just bc they are used in a couple places and might be easier to manage that way? totally not critical